packages/ui-extensions/src/surfaces/admin/components.d.ts

```diff
@@ -6281,6 +6284,38 @@ export interface AdminPrintActionJSXProps
   id?: string;
 }
 
+declare const tagName = 's-form';
```

Comment:
@vividviolet, should `Form`'s `tagName` export have a `$*` suffix like all of the other `tagName` exports?